packages/example/src/pages/components/code-blocks.mdx

```diff
@@ -56,3 +80,4 @@ This code snippet provides both a `path` and a `src`.
 | language | string   | [Available languages.](https://github.com/FormidableLabs/prism-react-renderer/blob/master/src/vendor/prism/includeLangs.js) |
 | src      | string   | The full url of a relevant link (source)                                                                                    |
 | path     | string   | A string indicating the filename or path. Due to markdown limitations this can only be a single word                        |
+| overflow | number   | A number indicating how many lines to display before overflowing. Use `0` to disable overflowing completely.                |
```

Comment:
My original purpose with this change was to be able to disable overflowing (it doesn't make sense in all contexts), making it configurable seemed good to me. That would enable an author to say "Well, if I have 10 lines, maybe I shouldn't overflow just one line".